### PR TITLE
th-202: Implement the 404 page according to the design

### PR DESCRIPTION
**Pull request by DmytroVoronkov**
- Added styles to not-found page

image

### CODE REVIEW

**Review by TetianaDoroshko — COMMENTED** [review]
Can we use already existing src/assets/img/auth-page-bg.svg? It contains tow-truck and 2 dashlines.

**Comment by DmytroVoronkov** [comment]
In given desing there was only 1 dashline, so i decided to create a new svg

**Comment by TetianaDoroshko** [comment]
I think it would be more consistently use the same bg.

File: frontend/src/pages/not-found/not-found.tsx [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
+import styles from './style.module.scss';
 
-    <Link to={AppRoute.ROOT}>
-      <span>Back to Home page</span>
+const NotFound = (): JSX.Element => (
+  <div className={styles.page}>
+    <h1 className={styles.code}>404</h1>
+    <p className={styles.message}>OOPS! PAGE NOT FOUND</p>
+    <Link className={styles.link} to={AppRoute.WELCOME}>
+      GO TO HOMEPAGE
```
Comment (Druxight) [inline frontend/src/pages/not-found/not-found.tsx]: Use the button component instead

**Review by Druxight — CHANGES_REQUESTED** [review]

**Review by TetianaDoroshko — APPROVED** [review]

**Review by h0wter — APPROVED** [review]

**Review by h0wter — APPROVED** [review]

File: frontend/src/pages/not-found/not-found.tsx [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
+import { Button } from '~/libs/components/components.js';
 import { AppRoute } from '~/libs/enums/enums.js';
+import { useCallback, useNavigate } from '~/libs/hooks/hooks.js';
 
-const NotFound = (): JSX.Element => (
-  <div>
-    <h2>
-      Oops, the dreaded <span>404</span>.
-    </h2>
+import styles from './style.module.scss';
```
Comment (kalinkaaaa14) [inline frontend/src/pages/not-found/not-found.tsx]:
```suggestion
import styles from './styles.module.scss';
```

File: frontend/src/pages/not-found/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
+@import "../../assets/css/vars.scss";
```
Comment (kalinkaaaa14) [inline frontend/src/pages/not-found/styles.module.scss]: src/assets/css/vars.scss

Comment (PavloSukhinin) [inline frontend/src/pages/not-found/styles.module.scss]:
```suggestion
@import "src/assets/css/vars.scss";
```

**Review by PavloSukhinin — DISMISSED** [review]

**Review by PavloSukhinin — APPROVED** [review]

**Pull request merged**